sources/packages/backend/apps/api/src/auth/_tests_/e2e/auth.e2e-spec.ts

```diff
@@ -254,6 +262,36 @@ describe("Authentication (e2e)", () => {
           expect(resp.body.email).toBeTruthy();
         });
     });
+
+    it(
+      "Should return a HttpStatus FORBIDDEN(403) when there is no user account associated to the user token " +
+        "to a default route that requires a user account.",
+      async () => {
+        await mockUserLoginInfo(moduleFixture, {
+          id: null,
+          user: { id: null, isActive: null } as User,
+        } as Student);
+        return request(app.getHttpServer())
+          .get("/auth-test/default-requires-user-route")
+          .auth(studentAccessToken, { type: "bearer" })
+          .expect(HttpStatus.FORBIDDEN);
```

Comment:
Even not being consistent in this file we have been validating the full error including the message in the `expect`.
If possible, I would recommend adding it to these new methods.